tests/ha/test_multi_server_pod_failure.py

```diff
@@ -285,3 +289,328 @@ def test_reads_kserver_pods_fail(self):
+        LOGGER.info("Step 6: Successfully performed DELETEs on random %s buckets", del_bucket)
+
+        LOGGER.info("Step 7: Perform READs on the remaining %s buckets.", remain_bkt)
+        rd_output = Queue()
+        new_s3data = {}
+        for bkt in remain_bkt:
+            new_s3data[bkt] = s3_data[bkt]
+        args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
+                'skipput': True, 'skipdel': True, 's3_data': new_s3data, 'di_check': True,
+                'output': rd_output}
+        self.ha_obj.put_get_delete(event, s3_test_obj, **args)
+        rd_resp = ()
+        while len(rd_resp) != 4:
+            rd_resp = rd_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        event_bkt_get = rd_resp[0]
+        fail_bkt_get = rd_resp[1]
+        event_di_bkt = rd_resp[2]
+        fail_di_bkt = rd_resp[3]
+
+        # Above four lists are expected to be empty as all pass expected
+        assert_utils.assert_false(len(fail_bkt_get) or len(fail_di_bkt) or len(event_bkt_get) or
+                                  len(event_di_bkt), "Expected pass in read and di check "
+                                                     "operations. Found failures in READ: "
+                                                     f"{fail_bkt_get} {event_bkt_get}"
+                                                     f"or DI_CHECK: {fail_di_bkt} {event_di_bkt}")
+        LOGGER.info("Step 7: Successfully performed READs on the remaining "
+                    "%s buckets.", remain_bkt)
+
+        LOGGER.info("ENDED: Test to verify DELETEs after %s (K) server pods down"
+                    " - unsafe shutdown.", self.kvalue)
+
+    # pylint: disable=too-many-statements
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-40578")
+    @CTFailOn(error_handler)
+    def test_writes_till_kserver_pods_fail(self):
+        """
+        Test to verify degraded WRITEs after each pod failure till K server pods
+        down - unsafe shutdown.
+        """
+        LOGGER.info("Started: Test to verify degraded Writes after each pod failure till K "
+                    "server pods down - unsafe shutdown.")
+
+        LOGGER.info("STEP 1: Perform WRITEs-READs-Verify with variable object sizes. 0B + "
+                    "(1KB - 512MB)")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-40578'
+        self.s3_clean = users
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix,
+                                                    skipcleanup=True, nsamples=2, nclients=2)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 1: Performed WRITEs-READs-Verify with variable sizes objects.")
+
+        LOGGER.info("Shutdown %s (K) server pods one by one and verify write/read/verify "
```

Comment:
We are verifying WRITEs only as part of this. READ/Validate is to verify WRITEs. Can we update the statement accordingly.